hack/install-controller-gen.sh

```diff
@@ -13,11 +14,11 @@ then
 fi
 
 
-CMDPATH="$GOPATH/pkg/mod/sigs.k8s.io/controller-tools@v0.4.0/cmd/controller-gen"
+CMDPATH="$GOPATH/pkg/mod/sigs.k8s.io/controller-tools@${CONTROLLER_TOOLS_VERSION}/cmd/controller-gen"
 
 if [ ! -f "$CMDPATH" ]
 then
     go mod download
 fi
 
-go build -o "${OUTPUT}" "$CMDPATH"
+go build -o "${OUTPUT}" "sigs.k8s.io/controller-tools/cmd/controller-gen"
```

Comment:
I guess we don't need a version here because we have the version pinned in go.mod?

Reply:
Yeah, that's the theory.
I don't know if it always works (i.e. gets the right version), but you can't specify a version to `go build`, only `go get`, according to the error message, so without this change it never works.